docs/docs/rasa-pro.mdx

```diff
@@ -44,6 +44,18 @@ Resolve performance issues faster and identify bottlenecks in message handling a
 
 [Read more here](./monitoring/tracing.mdx).
 
+### PII Handling
+
+Anonymize PII (Personal Identifiable Information) in logs and events streamed via the Kafka event broker. 
+
+[Read more here]
```

Comment:
ah basically we can't merge it until we have the links to the docs from Anca / Shailendra

Reply:
now I don't know how to add it in but I fished out the links: (./pii-management.mdx)